spec.bs

```diff
@@ -6395,7 +6450,13 @@ of the following global objects:
     1. Let (|result|, |executionMetrics|'s [=execution metrics/script timed out=]) be the
       result of [=evaluating a script=] with |realm|, |script|, |functionName|, |argumentsJS|,
       and |timeout|.
-    1. If |result| is an [=ECMAScript/abrupt completion=], return « "null", null, null, null ».
+    1. [=Finalize private aggregation debug info=] given |global|.
+    1. Let |paContributions| be the result of [=extracting private aggregation contributions=] given
+      |global|.
+    1. If |result| is an [=ECMAScript/abrupt completion=], return « "null", null, |paContributions|, null ».
+
+        Issue: Callers don't seem to expect nulls in the return value.

```

Comment:
Note: I updated the above line to pass on the private aggregation contributions, but the other fields might need updating too.

Reply:
yep, it needs to pass on the PAgg contributions.
1. The return array didn't have correct number of elements. See the last step of the algorithm, or callers of the algorithm for what's the expected return value. (it would be good if we have defined the return value format in the algorithm's declaration, but that was still a TODO).
2. It seems other fields can all be null, after checking the algorithm's callers

Reply:
1. Ah true -- updated to have two extra nulls
2. I think there is still an issue. In particular, |executionMetrics|'s [=script timed out=] is accessed by both callers, so |executionMetrics| being null isn't properly handled there (https://pr-preview.s3.amazonaws.com/alexmturner/turtledove/pull/1427.html#ref-for-evaluate-a-reporting-script and https://pr-preview.s3.amazonaws.com/alexmturner/turtledove/pull/1427.html#ref-for-evaluate-a-reporting-script%E2%91%A0). I guess that's not too major, though.